### PR TITLE
Feature/nijo 955 data provider to suppliers3 - Step3

### PR DESCRIPTION
**Pull request by mingazhev**

### CODE REVIEW

File: Api/Services/Connectors/SupplierConnectorManager.cs [hunk 1]
```diff
@@ -14,34 +14,34 @@
...
                 }
             };
         }
 
-        public IDataProvider Get(Suppliers key) => _dataProviders[key];
+        public ISupplierConnector Get(Suppliers key) => _dataProviders[key];
         
-        private readonly Dictionary<Suppliers, IDataProvider> _dataProviders;
-        private readonly DataProviderOptions _options;
+        private readonly Dictionary<Suppliers, ISupplierConnector> _dataProviders;
```
Comment (kirillta) [inline Api/Services/Connectors/SupplierConnectorManager.cs:44]: _dataProviders

File: Api/Services/Accommodations/Availability/AccommodationBookingSettingsService.cs [hunk 2]
```diff
@@ -19,11 +19,11 @@ public class AccommodationBookingSettingsService : IAccommodationBookingSettings
         public AccommodationBookingSettingsService(IDoubleFlow doubleFlow,
             IAgentSystemSettingsService agentSystemSettingsService,
             IAgencySystemSettingsService agencySystemSettingsService,
-            IOptions<DataProviderOptions> dataProviderOptions)
+            IOptions<SupplierOptions> dataProviderOptions)
```
Comment (kirillta) [inline Api/Services/Accommodations/Availability/AccommodationBookingSettingsService.cs:22]: dataProviderOptions

File: Api/Services/ProviderResponses/NetstormingResponseService.cs [hunk 3]
```diff
@@ -23,11 +23,11 @@ public class NetstormingResponseService
             IDistributedFlow flow,
             IBookingRecordsManager bookingRecordsManager,
             IBookingResponseProcessor responseProcessor,
-            IOptions<DataProviderOptions> dataProviderOptions,
+            IOptions<SupplierOptions> dataProviderOptions,
```
Comment (kirillta) [inline Api/Services/ProviderResponses/NetstormingResponseService.cs:26]: dataProviderOptions

**Review by kirillta — APPROVED** [review]

**Review by hasmikharutyunyan — APPROVED** [review]

File: Api/Services/Accommodations/Availability/AccommodationBookingSettingsService.cs (hunk 2 above)
Comment (mingazhev) [reply Api/Services/Accommodations/Availability/AccommodationBookingSettingsService.cs:22]: fixed in further requests

File: Api/Services/ProviderResponses/NetstormingResponseService.cs (hunk 3 above)
Comment (mingazhev) [reply Api/Services/ProviderResponses/NetstormingResponseService.cs:26]: fixed in further requests

File: Api/Services/Connectors/SupplierConnectorManager.cs (hunk 1 above)
Comment (mingazhev) [reply Api/Services/Connectors/SupplierConnectorManager.cs:44]: fixed in further requests

**Pull request merged**